website/docs/developer/research/research-localstack.md

```diff
@@ -59,6 +59,323 @@ Docker's localhost can be accessed on the parent machine at http://host.docker.i
     ```
     [Full example here.](https://git.cr.usgs.gov/LPDA/appeears/-/blob/feature/rds-setup/deployment/terraform/dev/main.tf)
 
+
+### Setup details
+
+-	Make sure you have Docker desktop installed in your laptop and approved by EROS helpdesk.
+- There are two ways to install localstack- either directly using docker command or by running a docker-compose.yaml file. The recommended way is by using docker compose since all changes will stay in one config file that can be changed according to use cases.
+#### Docker compose
+
+A sample docker-compose.yml file is shown below: 
+
```

Comment:
I would wrap this in a code block. See https://nasa.github.io/cumulus-orca/docs/developer/development-guide/documentation/documentation-style-guide#code

Since this is yaml, that is what I would use for a language.

```yaml
version: "3.8"

        services:
          localstack:
            container_name: localstack-container
            image: localstack/localstack
            ...
```